.github/workflows/release-stable.yml

```diff
@@ -3,20 +3,30 @@ name: Release Stable
 on: workflow_dispatch
 
 jobs:
-  bump-version:
-    runs-on: ubuntu-latest
+  release-stable:
+    name: Release stable
     timeout-minutes: 20
+
+    strategy:
+      matrix:
```

Comment:
Why matrix here?

Reply:
To ensure that it is set up the same way everywhere. If we want to make changes in the future or add more node versions or operating systems we can do this easily without going through the whole workflow job.

Reply:
I get the desire for symmetry but here its truly confusing because its a release workflow. It should not be using a matrix full stop.